src/accessible-date.js

```diff
@@ -79,19 +84,24 @@ module.exports = function accessibleDate(date, options) {
-            fr: [``, `une`, `deux`, `trois`, `quatre`, `cinq`, `six`, `sept`, `huit`, `neuf`, `dix`, `onze`, `douze`, `treize`, `quatorze`, `quinze`, `seize`, `dix-sept`, `dix-huit`, `dix-neuf`, `vingt`, `vingt et un`, `vingt-deux`, `vingt-trois`, `vingt-quatre`, `vingt-cinq`, `vingt-six`, `vingt-sept`, `vingt-huit`, `vingt-neuf`, `trente`, `Trente et un`, `Trente-deux`, `Trente-trois`, `Trente-quatre`, `Trente-cinq`, `Trente-six`, `Trente-sept`, `Trente-huit`, `Trente-neuf`, `quarante`, `quarante et un`, `quarante-deux`, `quarante-trois`, `quarante-quatre`, `quarante-cinq`, `quarante-six`, `quarante-sept`, `quarante-huit`, `quarante-neuf`, `cinquante`, `cinquante et un`, `cinquante-deux`, `cinquante-trois`, `cinquante-quatre`, `cinquante-cinq`, `cinquante-six`, `cinquante-sept`, `cinquante-huit`, `cinquante-neuf`]
+            fr: [``, `une`, `deux`, `trois`, `quatre`, `cinq`, `six`, `sept`, `huit`, `neuf`, `dix`, `onze`, `douze`, `treize`, `quatorze`, `quinze`, `seize`, `dix-sept`, `dix-huit`, `dix-neuf`, `vingt`, `vingt et un`, `vingt-deux`, `vingt-trois`, `vingt-quatre`, `vingt-cinq`, `vingt-six`, `vingt-sept`, `vingt-huit`, `vingt-neuf`, `trente`, `Trente et un`, `Trente-deux`, `Trente-trois`, `Trente-quatre`, `Trente-cinq`, `Trente-six`, `Trente-sept`, `Trente-huit`, `Trente-neuf`, `quarante`, `quarante et un`, `quarante-deux`, `quarante-trois`, `quarante-quatre`, `quarante-cinq`, `quarante-six`, `quarante-sept`, `quarante-huit`, `quarante-neuf`, `cinquante`, `cinquante et un`, `cinquante-deux`, `cinquante-trois`, `cinquante-quatre`, `cinquante-cinq`, `cinquante-six`, `cinquante-sept`, `cinquante-huit`, `cinquante-neuf`],
+            it: [`uno`, `due`, `tre`, `quattro`, `cinque`, `sei`, `sette`, `otto`, `nove`, `dieci`, `undici`, `dodici`, `tredici`, `quattordici`, `quindici`, `sedici`, `diciassette`, `diciotto`, `diciannove`, `venti`, `ventuno`, `ventidue`, `ventitre`, `ventiquattro`, `venticinque`, `ventisei`, `ventisette`, `ventotto`, `ventinove`, `trenta`, `trentuno`, `trentdue`, `trentatre`, `trentaquattro`, `trentacinque`, `trentasei`, `trentasette`, `trentotto`, `trentanove`, `quaranta`, `quarantuno`, `quarantadue`, `quarantatre`, `quarantaquattro`, `quarantacinque`, `quarantasei`, `quarantasette`, `quarantotto`, `quarantanove`, `quarantacinque`, `quarantasei`, `quarantasette`, `quarantotto`, `quarantanove`, `cinquanta`, `cinquantuno`, `cinquantadue`, `cinquantatre`, `cinquantaquattro`, `cinquantacinque` `cinquantasei`, `cinquantasette`, `cinquantotto`, `cinquantanove`],
```

Comment:
```suggestion
            it: [``, `uno`, `due`, `tre`, `quattro`, `cinque`, `sei`, `sette`, `otto`, `nove`, `dieci`, `undici`, `dodici`, `tredici`, `quattordici`, `quindici`, `sedici`, `diciassette`, `diciotto`, `diciannove`, `venti`, `ventuno`, `ventidue`, `ventitre`, `ventiquattro`, `venticinque`, `ventisei`, `ventisette`, `ventotto`, `ventinove`, `trenta`, `trentuno`, `trentdue`, `trentatre`, `trentaquattro`, `trentacinque`, `trentasei`, `trentasette`, `trentotto`, `trentanove`, `quaranta`, `quarantuno`, `quarantadue`, `quarantatre`, `quarantaquattro`, `quarantacinque`, `quarantasei`, `quarantasette`, `quarantotto`, `quarantanove`, `quarantacinque`, `quarantasei`, `quarantasette`, `quarantotto`, `quarantanove`, `cinquanta`, `cinquantuno`, `cinquantadue`, `cinquantatre`, `cinquantaquattro`, `cinquantacinque` `cinquantasei`, `cinquantasette`, `cinquantotto`, `cinquantanove`],
```

The first array item needs to be `0`.